addons/web_unsplash/__manifest__.py

```diff
@@ -15,13 +15,13 @@
         'web.assets_frontend': [
             'web_unsplash/static/src/js/unsplash_beacon.js',
         ],
+        'web_editor.assets_media_dialog_templates': [
```

Comment:
Could you add a `TODO` comment for those 2 new media dialog assets keys?
Something about what was discussed at https://github.com/odoo/odoo/pull/89223#discussion_r895627061 eg `TODO: Move those dialog assets in assets_common and make the public root fetch templates from the common asset`